src/typings/q/Q.d.ts

```diff
@@ -306,7 +306,7 @@ declare module Q {
     /**
      * A settable property that will intercept any uncaught errors that would otherwise be thrown in the next tick of the event loop, usually as a result of done. Can be useful for getting the full stack trace of an error in browsers, which is not usually possible with window.onerror.
      */
-    export var onerror: (reason: any) => void;
+    export var onerror: (reason: Error) => void;
```

Comment:
I just changed all the things that seemed reasonable that they could be an error.